main.tex

```diff
@@ -626,12 +626,24 @@ \subsection*{Prediction of resistance phenotypes for novel non-synonymous varian
 
 %%
 %
-...with the evidence for selection on Anopheles haplotypes, knowledge of the gene structure and evidence from other systems we can begin to predict resistance phenotypes for novel non-synonymous variants...
+Both indirect (clustering, networks) and direct evidence (EHH) for selection on \textit{Vgsc} haplotypes we detected, can be used in concert with knowledge of the gene's structure, linkage of variants, and evidence of phenotypes from both \textit{Anopheles} and other insect systems, to predict putative resistance phenotypes for the novel non-synonymous variants detected.
+%
+These predictions could help define relationships between newly detected variants and prioritise assay design for molecular tracking of variants/haplotypes in the field.
+%
+To explore this potential for predication, we divide the resistance phenotypes into known (experimentally shown) and putative, and divide the causative variants into phenotype drivers or enhancers.
+%
+Driver variants occur on haplotypes first, initiating the resistance phenotype and starting the selective sweep. 
+%
+Experimentally established resistance variants \texttt{L995F}, \texttt{L995S} in \textit{An. gambiae} and \texttt{V402L} in \textit{A. aegypti} (\textit{An. gambiae} codon numbering) are examples of known driver variants (Table \ref{table:variants_pheno}). 
+%
+Enhancer variants occur on haplotypes already undergoing selective sweeps due to driver variants, these enhancer variants either increase the resistant phenotype above that of the driver variants alone, or ameliorate fitness costs of carrying a driver. 
+%
+The \texttt{N1570Y} variant, which has only been found on haplotypes carrying \texttt{L995F} and, using \textit{Xenopus} oocytes, has been shown to produce an increased resistance phenotype only when paired with \texttt{L995F} \cite{Wang2015}, is thus classed as an enhancer variant (Table \ref{table:variants_pheno}). 
+%
 %%
-See Table 2
 
-@@TODO collect all material related to resistance phenotype prediction here, after we have results from analyses of selection.
 
+%we have described all the mutations in detail in the first part of the results section. I'm not sure what to add now to justify table 2 inclusion here? 
 
 %% Table 2 - Phenotype.
```

Comment:
I think we need to review a couple of things in this table. I think we need to make our domain nomenclature consistent with Wang 2015. Also we may want to nuance the "Phenotype" column. I'll give it some thought and post back.

Reply:
OK, here's an attempt at drafting something a bit more nuanced...

> \subsection*{Resistance phenotypes for novel non-synonymous variants}
> 
> %
> The sodium channel protein consists of four homologous domains (I-IV) each of which comprises six transmembrane segments (S1-S6) connected by intracellular and extracellular loops \cite{Dong2014}.
> %
> Two analogous pyrethroid binding sites have been predicted within the pore-forming modules of the protein, the first (PyR1) involving residues from transmembrane segments IIS5 and IIIS6 and the internal linker between IIS4 and IIS5 (IIL45) \cite{OReilly2006}, the second (PyR2) involving segments IS5, IS6, IIS6 and IL45 \cite{Du2013,Dong2014}.
> %
> Many of the amino acid substitutions known to cause pyrethroid resistance in insects affect residues within one of these two pyrethroid binding sites, and thus can directly alter pyrethroid binding \cite{Dong2014}.
> %
> For example, the \texttt{L995F} and \texttt{L995S} substitutions occur in segment IIS6 and belong to binding site PyR2 \cite{Du2013}.
> %
> The \texttt{I1527T} substitution that we discovered in \textit{An. coluzzii} mosquitoes from Burkina Faso occurs in segment IIIS6 and is immediately adjacent to two pyrethroid-sensing residues in site PyR1 \cite{Dong2014}.
> %
> It is thus plausible that pyrethroid binding could be altered by this substitution.
> %
> The I1527T substitution (M. domestica codon 1532) has been found in Aedes albopictus \cite{Xu2016}}, and multiple substitutions in codon 1529 (M. domestica codon 1534) have been reported in Aedes aegypti and associated with pyrethroid resistance \cite{Dong2014,Li2018}.
> %
> We found the I1527T allele in tight linkage with two alleles causing a V402L substitution (M. domestica codon 410), with haplotype structure indicating that an initial I1527T mutation was subsequently brought together with V402L alleles from different genetic backgrounds via recombination. 
> %
> Substitutions in codon 402 have been found in multiple insect species and are by themselves sufficient to confer pyrethroid resistance \cite{Dong2014}.
> %
> Codon 402 is within segment IS6, immediately adjacent to a pyrethroid sensing residue in Site PyR2.
> %
> The fact that we find I1527T and V402L in such tight association, with V402L apparently secondary to I1527T, is intriguing because (a) these two residues appear to affect different pyrethroid binding sites, and (b) haplotypes carrying V402L alone should also have been positively selected and thus be present in one or more populations.
> %%
> 
> %%
> %
> A number of substitutions in segments of the protein that are *not* involved either of the two pyrethroid binding sites have also been shown to confer pyrethroid resistance.
> %
> For example, the N1570Y substitution causes substantially enhanced pyrethroid resistance when combined with L995F, although codon 1570 occurs in the internal linker between domains III and IV (LIII/IV) \cite{Du2013}.
> %
> Computer modelling of the protein structure has suggested that substitutions in codon 1570 could allosterically alter site PyR2 and thus affect pyrethroid binding \cite{Du2013}.
> %
> In addition to N1570Y, we found thirteen other substitutions occurring exclusively in association with L995F at appreciable frequency (Table @@ref).
> %
> Of these, two (D466H, E1597G) occurred in the larger internal linkers between protein domains, one (R254K) occurred within a smaller internal linker between domain subunits, two (T791M, K1603T) occurred within an outer (“voltage-sensing”) transmembrane segment, one (A1746S) occurred within an inner (“pore-forming”) transmembrane segment, and the remaining seven occurred in the internal carboxyl-terminal tail.
> %
> Substitutions within various locations in the protein have been shown to confer pyrethroid resistance either independently or in combination with other substitutions not by altering pyrethroid binding but by altering the channel gating kinetics or the voltage-dependence of activation \cite{Dong2014}.
> %
> Thus there are a number of potential mechanisms by which a pyrethroid resistance phenotype can be obtained, and clearly much remains to be unravelled regarding the molecular biology of pyrethroid resistance in this gene.
> %%
> 

We may need to rething Table 2 a little also, can discuss tomorrow.

Reply:
I forgot to say something about M490I.